include/Ark/REPL/ConsoleStyle.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef CONSOLESTYLE_HPP
+#define CONSOLESTYLE_HPP
+
+#include <replxx.hxx>
+
+using Replxx = replxx::Replxx;
+
+namespace Ark
+{
+    const std::vector<std::string> KeywordsDict = {
```

Comment:
should be built using src/Builtins.cpp `const std::vector<std::pair<std::string, Value>> builtins;` variable as it has all the builtins, `operators` (same file) for operators and include/Ark/Parser/Lexer.hpp `const std::vector<std::string> keywords;` listing all the keywords